k3s_camera/main.py

```diff
@@ -0,0 +1,9 @@
+from time import sleep
+from picamera import PiCamera
+
+camera = PiCamera()
+camera.resolution = (1024, 768)
+camera.start_preview()
+# Warm up
```

Comment:
lol - what is the behavior if you capture before it's warmed up?

Reply:
It works fine, though I expect this is undefined behavior. This is just from the picamera docs:
https://picamera.readthedocs.io/en/release-1.13/recipes1.html